server/schemas/schemaRoutes.js

```diff
@@ -4,6 +4,12 @@ module.exports = function(router) {
   router.get("/", schemaController.getSchemaNames);
   router.get("/:schemaName", schemaController.getEntriesMetadataForSchema);
   router.get("/:schemaName/:userID", schemaController.getData);
+  /*
+  This is route is specifically for the download button.
+  Download requests don't let you set custom headers
+  so the jwt token has to be sent via url.
+  */
+  router.get("/:schemaName/:userID/:token", schemaController.getData);
```

Comment:
I kept line 6 because other functions call a get to "/:schemaName/:userID". I didn't want the other functions to have to put the token in the url.